understanding/conformance.html

```diff
@@ -642,7 +642,7 @@ <h3>Level of Assistive Technology Support Needed for "Accessibility Support"</h3
          
          <p>This topic raises the question of how many or which assistive technologies must support
             a Web technology in order for that Web technology to be considered "accessibility
-            supported". The WCAG Working group and the W3C do not specify which or how many assistive
+            supported". The AG Working Group and the W3C do not specify which or how many assistive
```

Comment:
Would we rather expand this to Accessibility Guidelines in all cases? (Other instances have been expanded from WCAG to Accessibility Guidelines in this PR, but this one is still abbreviated as AG.)

Reply:
I'd be in favour of expanding in all instances, yes

Reply:
```suggestion
            supported". The Accessibility Guidelines Working Group and the W3C do not specify which or how many assistive
```